scripts/kani-perf-setup.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# This is just the setup stage of all performance benchmarks
+# Other scripts should source this and invoke which setup they want to run
+
+set -o pipefail
+set -o nounset
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+KANI_DIR=$SCRIPT_DIR/..
+PERF_DIR="${KANI_DIR}/tests/perf"
+
+RUN_DIR="/tmp/kani_perf_run_$(date +%s)"
+
+build_kani() {
+# Build Kani using release mode.
+  cargo build-dev -- --release
+}
+
+prep_perf_files() {
+  mkdir ${RUN_DIR} # We want to error out if the folder exists
+  cp -r ${PERF_DIR} ${RUN_DIR}/perf
+  for overlay_dir in ${PERF_DIR}/overlays/*/; do
+    orig_dir=$(basename ${overlay_dir})
+    echo "Copying overlays for $orig_dir"
+    cp -r -v ${overlay_dir}* ${RUN_DIR}/perf/${orig_dir}/
+  done
+}
+
+cleanup_perf() {
+  echo "Cleaning up..."
+  rm -r ${RUN_DIR}
+  rm "$@"
+}
+
+run_benchmarks() {
+  suite=$(basename "${SUITE}")
+  mode="cargo-kani-test"
+  echo "Check compiletest suite=$suite mode=$mode"
+  cargo run -p compiletest -- --suite $suite --mode $mode --no-fail-fast --report-time "$@"
+}
+
+print_result() {
+  exit_code=$1
```

Comment:
Is this intentional?

Reply:
Yes. A bunch of these variables should be local, but I don't think it really matters here.